modules/lmdbbackend/lmdbbackend.cc

```diff
@@ -49,7 +49,7 @@
 
 #include "lmdbbackend.hh"
 
-#define SCHEMAVERSION 3
+#define SCHEMAVERSION 4
```

Comment:
do we need a migration? or are we good because you appended to the end of structs?

Reply:
The BOOST_CLASS_VERSION for DomainInfo is also increased. The migration is done on the fly based on the class version present in the database. https://github.com/PowerDNS/pdns/pull/11772/files#diff-c5a93f9a620612f89117271584c30d3b541e82f86cef733211d6f201c4d4a69cR209